Easy as abcDE: Piano Fingering Transcription Online

dvdrndlph opened the pull request:
Removed extraneous dot in Figure 2.

Commit:
Removed extraneous dot in Figure 2.

Commit:
Dactylize: Automatically Collecting Piano Fingering Data from Performance

Commit:
Missing email addresses.

Commit:
Removed references to old finger attachment design.

craffel commented:
Hi!  Sorry, I'm a little confused about the submission here - the title refers to your LBD paper which hs already been accepted, but it looks like you've added another paper ("Dactylize: Automatically Collecting...") and have made some changes to the Easy as abcDE paper.  Are you submitting the new Dactylize paper and some revisions to your old paper to review?

dvdrndlph commented:
Dear Colin:

Yes, I just removed a little splotch on a graphic and added an author's
email address in the first paper. Otherwise it is unchanged.

I am also submitting a second paper. Apparently anything I push to my repo
just gets appended to the active pull request. I didn't think it worked
that way. But the latest stuff there is the right stuff and ready for
review.

Sorry for the confusion.

Thanks,
Dave

On Tue, Jul 26, 2016 at 8:20 PM, Colin Raffel <notifications@github.com>
wrote:

> Hi! Sorry, I'm a little confused about the submission here - the title
> refers to your LBD paper which hs already been accepted, but it looks like
> you've added another paper ("Dactylize: Automatically Collecting...") and
> have made some changes to the Easy as abcDE paper. Are you submitting the
> new Dactylize paper and some revisions to your old paper to review?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ismir-net/ismir2016-lbd/pull/19#issuecomment-235454363>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEoMniKJEurtyHbNKY6X5hFjFRs43p05ks5qZrJggaJpZM4JVVU->
> .
>



-- 
David A. Randolph
Tuning Bell Studios LLC

Commit:
Removed extraneous word (*following*).

craffel commented:
Thank you for your second submission!  Very cool straightforward approach to solving this problem, and the paper does a good job of explaining it.  You may have one somewhere already, but I'd think a schematic would be useful.  Also, the link to the GitHub repository does not currently work, maybe you're planning on putting the code online at a later date?  Beyond those questions, the PDF is quite large - 12mb - for git's sake and for the sake of keeping the proceedings small-ish, is there any chance you can compress the images some more to lower the PDF size?  If you have Acrobat Pro, it has some functionality built-in for doing this sort of thing.  Thank you again!

dvdrndlph commented:
I plan to include a [Fritzing diagram](http://dvdrndlph.github.io/didactyl/img/fritzing.png) of the circuit on the poster. This is all I have by way of a schematic. There wasn't enough room in the paper for this. I could maybe trade this diagram for the architecture diagram. Or just provide a link?

The Git repo was private. That is why you couldn't get to it. This is fixed now. Thanks for the heads up.

I will investigate how to shrink the size of this thing. Not sure what is going on there. The whole seems larger than the sum of parts.

Thanks,
Dave

Commit:
Reduce size of randolph_dactylize.pdf.

dvdrndlph commented:
I have reduced the size of the PDF, but left the Fritzing diagram out. I think the high-level picture is more important.

Commit:
Made space for Fritzing diagram below condensed architecture diagram.

dvdrndlph commented:
I have tightened some text, shrunk the architecture diagram, and pried open enough space for a reformatted Fritzing diagram. This should give anyone trying to reproduce this system an important piece of the puzzle. Thanks for nudging me to do this.

Commit:
Centered architecture graphic in column.

craffel commented:
Awesome, thank you, merging!
